### PR TITLE
Shop page

### PR DESCRIPTION
**Pull request by HaydenBS93**

### CODE REVIEW

File: shop.html [hunk 1]
```diff
@@ -0,0 +1,599 @@
...
+          >
+          <a
+            href="#"
+            class="text-travel-orange text-lg font-semibold hover:text-orange-600"
+            >Blog</a
+          >
+          <a
+            href="#"
+            class="text-travel-orange text-lg font-semibold hover:text-orange-600"
+            >Contact</a
```
Comment (naha92) [inline shop.html:54]: Missing links

File: shop.html [hunk 2]
```diff
@@ -0,0 +1,599 @@
...
+    <main>
+      <section class="max-w-screen-d mx-auto py-12 px-8">
+        <h1 class="py-12 text-5xl text-travel-orange font-medium text-center">
+          Travel Shop
+        </h1>
+        <h2 class="pb-4 text-3xl font-semibold">Collections</h2>
+        <div
+          class="grid grid-cols-1 gap-x-6 gap-y-10 sm:grid-cols-2 lg:grid-cols-3 xl:gap-x-8"
+        >
+          <a href="#" class="group">
```
Comment (naha92) [inline shop.html:80]: Missing link

File: shop.html [hunk 3]
```diff
@@ -0,0 +1,599 @@
...
+              Most of our products are limited releases that won't come back.
+              Get your favorite items while they're in stock
+            </p>
+          </div>
+          <div class="mt-4">
+            <button
+              class="px-4 py-2 bg-travel-orange text-white font-semibold rounded-lg hover:bg-orange-600"
+            >
+              Get access to our one-time sale
+            </button>
```
Comment (naha92) [inline shop.html:165]: No link on button - maybe link to contact page?

File: shop.html [hunk 4]
```diff
@@ -0,0 +1,599 @@
...
+                  />
+                </div>
+                <h3 class="mt-4 text-travel-orange text-lg font-semibold">
+                  Bose Headphones
+                </h3>
+              </a>
+              <p class="mt-1 text-sm">
+                Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum
+              </p>
+            </div>
```
Comment (naha92) [inline shop.html:233]: Missing alt tags, missing links (maybe link to external page of the product?)

File: shop.html [hunk 5]
```diff
@@ -0,0 +1,599 @@
...
+          </div>
+          <div class="flex justify-end items-center mt-4">
+            <button
+              class="text-travel-orange font-semibold hover:text-orange-600"
+            >
+              More Travel Tech &rarr;
+            </button>
```
Comment (naha92) [inline shop.html:240]: Missing link on button

File: shop.html [hunk 6]
```diff
@@ -0,0 +1,599 @@
...
+              <p class="mt-1 text-sm">
+                Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum
+              </p>
+            </div>
+          </div>
+          <div class="flex justify-end items-center mt-4">
+            <button
+              class="text-travel-orange font-semibold hover:text-orange-600"
+            >
+              More Accessories &rarr;
```
Comment (naha92) [inline shop.html:312]: Missing link on button

File: shop.html [hunk 7]
```diff
@@ -0,0 +1,599 @@
...
+                    alt=""
+                    class="w-full h-96 object-cover object-center"
+                  />
+                </div>
+                <h3 class="mt-4 text-travel-orange text-lg font-semibold">
+                  Kiels Hand Sanitiser
+                </h3>
+              </a>
+              <p class="mt-1 text-sm">
+                Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum
```
Comment (naha92) [inline shop.html:377]: Missing alt tags, missing links (maybe link to external page of the product?)

File: shop.html [hunk 8]
```diff
@@ -0,0 +1,599 @@
...
+              </p>
+            </div>
+          </div>
+          <div class="flex justify-end items-center mt-4">
+            <button
+              class="text-travel-orange font-semibold hover:text-orange-600"
+            >
+              More Health &amp; Beauty &rarr;
```
Comment (naha92) [inline shop.html:385]: Missing link on button

File: shop.html [hunk 9]
```diff
@@ -0,0 +1,599 @@
...
+            href="#"
+            class="text-lg font-semibold text-travel-orange hover:text-orange-600"
+            >Blog</a
+          >
+        </div>
+        <div class="pb-6">
+          <a
+            href="#"
+            class="text-lg font-semibold text-travel-orange hover:text-orange-600"
+            >Contact</a
```
Comment (naha92) [inline shop.html:563]: missing links

File: scripts/scripts.js [hunk 10]
```diff
@@ -28,28 +28,3 @@ function topFunction() {
...
-  }
-
-  acceptCookiesButton.addEventListener("click", hideCookieBanner);
-  rejectCookiesButton.addEventListener("click", hideCookieBanner);
-
-  setTimeout(function () {
-    // Add a class to make the banner visible after 5 seconds
-    cookieBanner.classList.add("opacity-100");
-  }, 5000);
-});
```
Comment (naha92) [inline scripts/scripts.js:55]: Why removing cookie banner?

**Review by naha92 — CHANGES_REQUESTED** [review]
Added comments to shop.html and scripts.js

File: shop.html (hunk 1 above)
Comment (HaydenBS93) [reply shop.html:54]: done

File: shop.html [hunk 11]
```diff
@@ -0,0 +1,599 @@
...
+              <h3 class="text-travel-orange text-xl font-semibold">
+                Accessories
+              </h3>
+            </div>
+            <p class="mt-1 text-sm font-medium">
+              Explore our accessory section for the perfect finishing touch to
+              elevate your style
+            </p>
+          </a>
+          <a href="#" class="group">
```
[the remark this answers is not in the document]
Comment (HaydenBS93) [reply shop.html:120]: done

File: shop.html (hunk 3 above)
Comment (HaydenBS93) [reply shop.html:165]: this is just a dummy button as we dont have any sales as its just a static site

File: shop.html (hunk 5 above)
Comment (HaydenBS93) [reply shop.html:240]: Leaving as a dummy button as no link to a different page for all products

File: shop.html (hunk 6 above)
Comment (HaydenBS93) [reply shop.html:312]: Leaving as a dummy button as no link to a different page for all products

File: shop.html (hunk 8 above)
Comment (HaydenBS93) [reply shop.html:385]: Leaving as a dummy button as no link to a different page for all products

File: shop.html (hunk 7 above)
Comment (HaydenBS93) [reply shop.html:377]: done alt tags and also changed product names too as these were duplicated
Just leaving the links as no endpoint as no other way to open the product out, just static placement

File: shop.html (hunk 9 above)
Comment (HaydenBS93) [reply shop.html:563]: done

File: scripts/scripts.js (hunk 10 above)
Comment (HaydenBS93) [reply scripts/scripts.js:55]: i forgot to add back in when i was cleaning up some code

File: shop.html [hunk 12]
```diff
@@ -0,0 +1,599 @@
...
+              </p>
+            </div>
+          </div>
+          <div class="sm:flex items-center">
+            <div class="sm:flex-shrink-0">
+              <div class="flow-root">
+                <img
+                  class="h-16 w-16"
+                  src="assets/icons/icons8-returns-64.png"
+                  alt=""
```
Comment (naha92) [inline shop.html:442]: Alt tag

File: shop.html [hunk 13]
```diff
@@ -0,0 +1,599 @@
...
+              </p>
+            </div>
+          </div>
+          <div class="sm:flex items-center">
+            <div class="sm:flex-shrink-0">
+              <div class="flow-root">
+                <img
+                  class="h-12 w-12"
+                  src="assets/icons/icons8-sustainability-50.png"
+                  alt=""
```
Comment (naha92) [inline shop.html:463]: Alt tag

File: shop.html [hunk 14]
```diff
@@ -0,0 +1,599 @@
...
+          <span class="sr-only">YouTube</span>
+          <img src="/assets/icons/youtube.svg" alt="Youtube icon" />
+        </a>
+        <a href="https://www.instagram.com/">
+          <span class="sr-only">Instagram</span>
+          <img src="/assets/icons/instagram.svg" alt="Instagram icon" />
+        </a>
+      </div>
+      <p class="mt-10 text-center text-lg text-black">
+        &copy; 2020 TravelUp, Inc. All rights reserved.
```
Comment (naha92) [inline shop.html:594]: 2024

**Review by naha92 — CHANGES_REQUESTED** [review]
Changes look good, caught a couple minor things I missed on first review.

File: shop.html [hunk 15]
```diff
@@ -0,0 +1,599 @@
...
+              </p>
+            </div>
+          </div>
+          <div class="sm:flex items-center">
+            <div class="sm:flex-shrink-0">
+              <div class="flow-root">
+                <img
+                  class="h-16 w-16"
+                  src="assets/icons/icons8-warranty-80.png"
+                  alt=""
```
Comment (naha92) [inline shop.html:421]: Alt tag

File: shop.html (hunk 12 above)
Comment (HaydenBS93) [reply shop.html:442]: done

File: shop.html (hunk 15 above)
Comment (HaydenBS93) [reply shop.html:421]: done

**Review by naha92 — APPROVED** [review]

**Pull request merged**